src/diffusers/models/lora.py

```diff
@@ -168,7 +168,6 @@ def _fuse_lora(self):
             return
 
         dtype, device = self.weight.data.dtype, self.weight.data.device
-        logger.info(f"Fusing LoRA weights for {self.__class__}")
```

Comment:
The problem with "info" here is that it would pop up 100+ times for models like SDXL which makes it unreadable. Think we should better remove it.